jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
```

Comment:
After I verify that using `get-kube.sh` works, L133 - L194 and L353 - L370 go away, as do all of the conditionals using `JENKINS_USE_GET_KUBE_SCRIPT`.

I'm duplicating the existing logic, guarded by this environment variable, since there's no better way to test Jenkins changes incrementally.

There will be less logic in this script when we're all done.